hive_metastore_client/hive_metastore_client.py

```diff
@@ -212,3 +212,29 @@ def _validate_lists_length(list_a: List[Any], list_b: List[Any]) -> None:
                 "m=_validate_lists_length, msg=The length of the two provided "
                 "lists does not match"
             )
+
+    def get_partition_keys(self, db_name: str, table_name: str) -> List[FieldSchema]:
```

Comment:
since `year` and `2021` are partitions keys, what if we stated that `2021` is a partition key **value**, just as you did for the partition key **name** `year`?
or if you want to keep as _partition key_ for the values, maybe dropping the **key** from _partition key name_ could work

Reply:
@felipemiquelim I'm not quite sure, but I think @ribaldorafael  misunderstood the goal of the method and understood that this method returns the partition value, when in fact this method returns the partition keys, but as a FieldSchema object (not only the string names). I reenforce what I commented [here](https://github.com/quintoandar/hive-metastore-client/pull/43#discussion_r570283873)  then.

Reply:
@ribaldorafael as Lucas said, partition-key is always the partition column itself; Thrift already provides us all the methods for accessing the partition values directly (`get_partition`, `get_partitions`, `get_partition_by_name`) but it does not provide any access directly to the partition-keys (the columns defined as partitions in the table schema).
I will follow @LucasMMota suggestion and rename the first method to `get_table_partition_keys_definition` so we can make things more intuitive